### PR TITLE
stats: add metircs for high error rate feedback

### PR DESCRIPTION
**Pull request by winoros**
### What problem does this PR solve? 

Add metrics to count the case when the actual count got by feedback is much different from the expected count calculated by statistics.

### Check List 

Tests 

 - No code

Related changes

 - Need to update the `tidb-ansible` repository

### CODE REVIEW

File: metrics/stats.go [hunk 1]
```diff
@@ -76,4 +76,12 @@ var (
 			Name:      "store_query_feedback_total",
 			Help:      "Counter of storing query feedback.",
 		}, []string{LblType})
+
+	HighErrorRateFeedBackCounter = prometheus.NewCounter(
```
Comment (eurekaka) [inline metrics/stats.go]: s/HighErrorRateFeedBackCounter/SignificantFeedbackCounter?

File: statistics/update.go [hunk 2]
```diff
@@ -193,8 +193,11 @@ func (s *SessionStatsCollector) StoreQueryFeedback(feedback interface{}, h *Hand
 	} else {
 		rate = math.Abs(expected-float64(q.actual)) / float64(q.actual)
 	}
-	if rate >= MinLogErrorRate && (q.actual >= MinLogScanCount || q.expected >= MinLogScanCount) && log.GetLevel() == log.DebugLevel {
-		q.logDetailedInfo(h)
+	if rate >= MinLogErrorRate && (q.actual >= MinLogScanCount || q.expected >= MinLogScanCount) {
+		metrics.HighErrorRateFeedBackCounter.Inc()
+		if log.GetLevel() == log.DebugLevel {
```
Comment (zz-jason) [inline statistics/update.go]: Can we change this log level to INFO or WARN?

File: metrics/stats.go [hunk 3]
```diff
@@ -76,4 +76,12 @@ var (
...
+		prometheus.CounterOpts{
+			Namespace: "tidb",
+			Subsystem: "statistics",
+			Name:      "high_error_rate_feedback_total",
```
[the remark this answers is not in the document]
Comment (winoros) [reply metrics/stats.go]: Yes.

File: statistics/update.go (hunk 2 above)
Comment (winoros) [reply statistics/update.go]: This part logs many things and has some heavy calculation. So it would be better to set it to debug level.

**Pull request merged**